Makefile

```diff
@@ -270,7 +274,10 @@ deploy-onprem-for-subsystem:
 	export DUMMY_IGNITION="true" && $(MAKE) deploy-onprem
 
 deploy-on-openshift-ci:
-	echo "placeholder for deployment on openshift ci"
+	ln -s $(shell which oc) $(shell dirname $(shell which oc))/kubectl
+	export TARGET='oc' && export PROFILE='openshift-ci' && unset GOFLAGS && \
+	$(MAKE) jenkins-deploy-for-subsystem
```

Comment:
`deploy-on-openshift-ci` shouldn't call `jenkins-...`
Please change the naming

Reply:
Just to make sure I understand correctly, you want the existing jenkins-deploy-for-subsystem target renamed. Correct?

Reply:
It's an option. But in that case, you would break the Jenkins pipeline. (Unless you would change the Jenkinsfile content as well).
On another hand, you can create another target that both `jenkins-..` and `-openshift-ci-..` targets would call.

IMO, I am against referring the CIs... The repository shouldn't know which entity is running itself.

Reply:
As you recall, on the call last week it was implemented with two different targets - one for jenkins and one for openshift-ci 
 with very little difference (you asked about it then :) ). I'm ok with whichever approach you prefer - either to rename the target (changing the Jenkinsfile content) or duplicating the target. Let us know which one you prefer.

Reply:
I'd prefer having one target for CI :)

Reply:
Will do!